packages/back-end/src/events/notification-events.d.ts

```diff
@@ -30,6 +30,7 @@ export type FeatureUpdatedNotificationEvent = NotificationEventPayload<
   {
     current: ApiFeature;
     previous: ApiFeature;
+    updatedBy: string;
```

Comment:
We don't need this field in the event payload.  Events already have the `user` who triggered the event.

Reply:
Thanks!